datadog-notifier-console-plugin-l10n/pom.xml

```diff
@@ -44,11 +44,11 @@
     <parent>
         <groupId>fish.payara.extensions.notifiers</groupId>
         <artifactId>notifiers-parent</artifactId>
-        <version>1.1-enterprise-SNAPSHOT</version>
+        <version>1.0.1-enterprise</version>
```

Comment:
Is this version correct? `1.0.1-enterprise`  and not `1.1-enterprise`?

Reply:
Is that from commit https://github.com/payara/Notifiers/pull/22/commits/c0a7ab5c99ecece8a1d1adc2ead059bc5173ed41? If so, yes as the notifiers are being individually versioned but in this case should use the same parent pom (1.0.1-enterprise) as nothing changed, hence I needed to essentially revert 51aaedc8ed2530c97d9fb7d9c1a3cf8a0b8972de so the project would compile and I can reference parent pom 1.0.1-enterprise for the release of the teams-notifier-console-plugin